wled00/data/index.js

```diff
@@ -68,8 +68,8 @@ function applyCfg()
 	var ccfg = cfg.comp.colors;
 	d.getElementById('hexw').style.display = ccfg.hex ? "block":"none";
 	d.getElementById('picker').style.display = ccfg.picker ? "block":"none";
-  d.getElementById('vwrap').style.display = ccfg.picker ? "block":"none";
-  d.getElementById('kwrap').style.display = ccfg.picker ? "block":"none";
+	d.getElementById('vwrap').style.display = ccfg.picker ? "block":"none";
```

Comment:
Thanks!
These indentations were indeed horrible _if you use an tab size >2_. I prefer tab size 2 for my editor, since it makes indentations more compact. Github uses a default of 8 (!), so my code rendered fine in _my_ VS Code, but horribly in the web view and (I assume) VS Code with tab size 4.
I had the option enabled though in VS Code which inserted 2 spaces instead of a tab, which is bad, since it essentially forces tab size 2 upon everyone, but only for the lines I edited recently. Turned that off now, we should replace all indentations with spaces with tab indentations. That way indentations should render correctly no matter what tab size you prefer.

Reply:
https://softwareengineering.stackexchange.com/questions/57/tabs-versus-spaces-what-is-the-proper-indentation-character-for-everything-in-e 😅 
No "100% correct" way as it seems.
Let's go with the style proposed by Peter Boughton:

_Use tabs to indent the start of the line, one tab per indent level, and let everyone pick how wide they want that to be._

_Use spaces if you're lining up characters within a line, so they always line up regardless of tab size._